webhook/webhook.py

```diff
@@ -75,66 +88,134 @@ def handle_team_event(self, payload):
-                          format(str(payload)))
+            logging.debug(f"team edited event triggered: {str(payload)}")
             try:
                 team = self.__facade.retrieve(Team, github_id)
                 team.github_team_name = github_team_name
-                logging.info("changed team's name with id {} from {} to {}".
-                             format(github_id, github_team_name,
-                                    team.github_team_name))
+                logging.info(f"changed team's name with id {github_id} from "
+                             f"{github_team_name} to {team.github_team_name}")
                 self.__facade.store(team)
-                logging.info("updated team with id {} in rocket db."
-                             .format(github_id))
-                return "updated team with id {}".format(github_id), 200
+                logging.info(f"updated team with id {github_id} in"
+                             " rocket db.")
+                return f"updated team with id {github_id}", 200
             except LookupError:
-                logging.error("team with github id {} not found.".
-                              format(github_id))
-                return "team with github id {} not found"\
-                       .format(github_id), 404
+                logging.error(f"team with github id {github_id} not found.")
+                return f"team with github id {github_id} not found", 404
         elif action == "added_to_repository":
             repository_name = payload["repository"]["name"]
-            logging.info("team with id {} added to repository {}"
-                         .format(github_id, repository_name))
-            return "team with id {} added to repository {}"\
-                   .format(github_id, repository_name), 200
+            logging.info(f"team with id {github_id} added to repository"
+                         f" {repository_name}")
+            return (f"team with id {github_id} added to repository"
+                    f" {repository_name}", 200)
         elif action == "removed_from_repository":
             repository_name = payload["repository"]["name"]
-            logging.info("team with id {} from repository {}"
-                         .format(github_id, repository_name))
-            return "team with id {} removed repository {}"\
-                   .format(github_id, repository_name), 200
+            logging.info(f"team with id {github_id} from repository"
+                         f" {repository_name}")
+            return (f"team with id {github_id} removed repository "
+                    f"{repository_name}", 200)
         else:
-            logging.error("invalid payload received: {}".
-                          format(str(payload)))
+            logging.error(f"invalid payload received: {str(payload)}")
             return "invalid payload", 405
+
+    def handle_membership_event(self, payload):

```

Comment:
The actual endpoint for this will have to be added to `server.py`. However I think you can do that as part of your work on #260.